copy_peste.yml

```diff
@@ -3,11 +3,11 @@ environment: "development"
 development:
   modules:
       graphics:
-          default: "console_mode/console_mode.rb"
+          default: "console_mode"
   ports_tree_path: "./"
 
 production:
   modules:
       graphics:
-          default: "console_mode/console_mode.rb"
+          default: "console_mode"
   ports_tree_path: "/"
```

Comment:
+1, however, it stays customizable throw the command line (cf my previous feature)